pandas/tests/arrays/categorical/test_constructors.py

```diff
@@ -458,6 +458,14 @@ def test_constructor_with_categorical_categories(self):
         result = Categorical(["a", "b"], categories=CategoricalIndex(["a", "b", "c"]))
         tm.assert_categorical_equal(result, expected)
 
+    def test_construction_with_na(self):
```

Comment:
If you rename test to `test_construction_with_null` and use the `nulls_fixture` I think would be more comprehensive (will have to skip pd.NaT)